src/components/Patient/PatientHome.tsx

```diff
@@ -65,6 +65,12 @@ export const PatientHome = (props: {
     AssignedToObjectModel | undefined
   >(patientData.assigned_to_object);
 
+  const [isPatientHomeMounted, setIsPatientHomeMounted] = useState(false);
+
+  useEffect(() => {
+    setIsPatientHomeMounted(true);
+  }, []);
```

Comment:
what does this do?

Reply:
It ensures PatientHome to render first. Because before rendering of patientHome , demography tab is rendering